src/lib/tvgPicture.cpp

```diff
@@ -98,6 +98,7 @@ Result Picture::size(float w, float h) noexcept
 Result Picture::size(float* w, float* h) const noexcept
 {
     if (!pImpl->loader) return Result::InsufficientCondition;
+    pImpl->reload();
```

Comment:
Should reload() called on size() and viewbox() getters?
Is there any overhead from this?

If the issue is the size when the user calls load() and immediately uses the size(), viewbox() getter
It seems that we can call resize() after reading in load() function.

Reply:
yes, if a user calls size/viewbox after load in a 'normal' case the information are already available, since header() was already called. but for svgs without width/height and viewbox info, this information is available after run() is finished, that's why I need to call reload here, to be sure that run is done.